pyproject.toml

```diff
@@ -1,20 +1,100 @@
+[project]
+name = "SimpleLogin"
+version = "0.1.0"
+description = "SimpleLogin partner API"
+authors = [ {name="SimpleLogin", email="dev@simplelogin.io"}]
+license = "MIT"
+repository = "https://github.com/simple-login/app"
+keywords = ["email", "alias", "privacy", "oauth2", "openid"]
+packages = [
+    { include = "app/" },
+    { include = "migrations/" },
+]
+include = ["templates/*", "templates/**/*", "local_data/*.txt"]
+
+requires-python = "~=3.10"
+
+dependencies = [
```

Comment:
is there a way to move this one at the same location as `[tool.poetry.dependencies]` and `[tool.poetry]` to faciliate git diff comparison?

Reply:
The issue is that poetry uses their own non-standard way of defining dependencies. Rye follows the standard convention for python dependency management with the standard format.

Reply:
We basically copied the poetry ones and replaced the format to the new one.